From 0fadf0a3baee106b0eb12c32f9e49140c5355221 Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@gmail.com>
Date: Fri, 2 Sep 2016 01:18:20 -0700
Subject: [PATCH] DOCKER-923 sdc-docker api-create tests checks for incorrect
 property on error object when determining whether fabrics are enabled

---
 test/integration/api-create.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/integration/api-create.test.js b/test/integration/api-create.test.js
index da3c40f..a98b14c 100644
--- a/test/integration/api-create.test.js
+++ b/test/integration/api-create.test.js
@@ -353,7 +353,7 @@ test('ensure fabrics enabled', function (tt) {
             function (err, vlans) {
                 if (err) {
                     FABRICS = false;
-                    if (err.code !== 'PreconditionRequiredError') {
+                    if (err.restCode !== 'PreconditionRequiredError') {
                         t.ifErr(err);
                     }
                 } else {
-- 
2.21.0

